_data/social.yml

```diff
@@ -216,4 +216,4 @@ websites:
       img: zocle.png
       tfa: Yes
       sms: Yes
-      doc: http://university.zocle.com/2015/03/30/new-security-feature-two-step-verification/
+      doc: http://university.zocle.com/2015/03/30/new-security-feature-two-step-verification/
```

Comment:
Was this requirement explicitly stated in readme.md? No. Contributing.md? No. Was it stated in either one of those files that we should be consulting .editorconfig for guidance on formatting our code? No.

So lose your attitude. It is completely illogical to suggest that I should "respect" a rule that I did not even know existed.